### PR TITLE
feat(employee standing) change currency

### PR DESCRIPTION
**Pull request by jeremielodi**

### CODE REVIEW

File: server/controllers/finance/creditors.js [hunk 1]
```diff
@@ -104,44 +107,58 @@ function balance(creditorUuid) {
...
+
       WHERE g.entity_uuid = ?
       GROUP BY g.record_uuid
     )c, (SELECT @cumsum := 0)z
     ORDER BY trans_date ASC, trans_id;
   `;
 
   return q.all([
-    db.exec(sql, [uid, uid]),
-    balance(creditorUuid),
```
Comment (jniles) [inline server/controllers/finance/creditors.js:137]: What happened to `balance`? 

It seems like we might have rewritten SQL code into JS and replaced `balance()` with `_.sumBy()`?  If so, that is the opposite of what we want to do.  JS struggles with math, so let's make whatever changes necessary to keep the sums in SQL.

File: server/controllers/finance/reports/financial.employee.handlebars [hunk 2]
```diff
@@ -57,13 +57,13 @@
...
-            <span>{{currency aggregates.debit metadata.enterprise.currency_id}}</span>
+            <span>{{currency aggregates.debit ./currency_id}}</span>
           </th>
           <th class="text-right">
-            <span>{{currency aggregates.credit metadata.enterprise.currency_id}}</span>
+            <span>{{currency aggregates.credit ./currency_id}}</span>
           </th>
           <th class="text-right">
-            <span>{{currency aggregates.balance metadata.enterprise.currency_id}}</span>
+            <span>{{debcred aggregates.balance ./currency_id}}</span>
```
Comment (jniles) [inline server/controllers/finance/reports/financial.employee.handlebars]: 👍

**Comment by jeremielodi** [comment]
@jniles this PR is ready for a review

**Comment by jniles** [comment]
@jeremielodi, the code looks better now.

I don't think it makes sense to change the debits and the credits to be in a different currency ... this is a record of what actually happened in the transaction.  A better way is to do what the account statement does with multiple currencies:
accountstatement


As you can see, the debit/credit columns show the original transaction amount, but we add two columns for the exchange rate and the converted value.  This will allow a user to see both what the original amount was (what actually happened) and then what the converted amount is.

The totals are right - they should be converted into the selected currency.  Well done there!

**Comment by jniles** [comment]
@jeremielodi, what is the status of this PR?

**Comment by jeremielodi** [comment]
I'm working on it

**Comment by jniles** [comment]
Looks like this will need to be updated to reflect the changes in https://github.com/IMA-WorldHealth/bhima/pull/3596.

**Comment by kwilu** [comment]
Heads up @jeremielodi, we just merged some commits that conflict with the changes your made in this pull request! You can review this repository's [recent commits](https://github.com/IMA-WorldHealth/bhima/commits/master) to see where the conflicts occur. Please rebase your feature branch against the `upstream/master` branch and [resolve](https://zulip.readthedocs.io/en/latest/git/troubleshooting.html#recover-from-a-git-rebase-failure) your pull request's merge conflicts accordingly.

**Comment by kwilu** [comment]
Heads up @jeremielodi, we just merged some commits that conflict with the changes your made in this pull request! You can review this repository's [recent commits](https://github.com/IMA-WorldHealth/bhima/commits/master) to see where the conflicts occur. Please rebase your feature branch against the `upstream/master` branch and [resolve](https://zulip.readthedocs.io/en/latest/git/troubleshooting.html#recover-from-a-git-rebase-failure) your pull request's merge conflicts accordingly.

**Comment by kwilu** [comment]
Heads up @jeremielodi, we just merged some commits that conflict with the changes your made in this pull request! You can review this repository's [recent commits](https://github.com/IMA-WorldHealth/bhima/commits/master) to see where the conflicts occur. Please rebase your feature branch against the `upstream/master` branch and [resolve](https://zulip.readthedocs.io/en/latest/git/troubleshooting.html#recover-from-a-git-rebase-failure) your pull request's merge conflicts accordingly.

**Pull request closed without merging**